sdk/eventhub/Microsoft.Azure.EventHubs/tests/Client/TokenProviderTests.cs

```diff
@@ -124,13 +124,20 @@ public async Task UseITokenProviderWithAad()
                 return;
             }
 
-            var authContext = new AuthenticationContext($"https://login.windows.net/{tenantId}");
-            var cc = new ClientCredential(aadAppId, aadAppSecret);
-            var tokenProvider = TokenProvider.CreateAadTokenProvider(authContext, cc);
+            AzureActiveDirectoryTokenProvider.AuthenticationCallback authCallback = 
+                async (audience, authority, state) =>
+                {
+                    var authContext = new AuthenticationContext(authority);
+                    var cc = new ClientCredential(aadAppId, aadAppSecret);
```

Comment:
>var cc = new ClientCredential(aadAppId, aadAppSecret); [](start = 20, length = 54)

Is this test supposed to pass when run as is? When I do the same here using `new ClientCredetial("", "")` I get ArgumentException.